src/components/CreateMeritRequest.svelte

```diff
@@ -100,7 +98,7 @@
 			<Dialog.Title>Request Merits</Dialog.Title>
 			<Dialog.Description>Request Merits for your work</Dialog.Description>
 		</Dialog.Header>
-		<div class="grid gap-4 py-4">
+		<div class="grid gap-4 overflow-auto py-4">
```

Comment:
The actual modification consists of only two lines; the other lines are changes resulting from formatting with Prettier.